tests/gis_tests/tests.py

```diff
@@ -81,3 +89,11 @@ def test_no_version_number(self):
         ops = FakePostGISOperations()
         with self.assertRaises(ImproperlyConfigured):
             ops.spatial_version
+
+
+class TestGeometryColumns(SimpleTestCase):
+
+    def test_geom_col_name(self):
+        self.assertEqual(OracleGeometryColumns.geom_col_name(), 'column_name')
+        self.assertEqual(PostGISGeometryColumns.geom_col_name(), 'f_geometry_column')
+        self.assertEqual(SpatialiteGeometryColumns.geom_col_name(), 'f_geometry_column')
```

Comment:
I would prefer to use `connection.ops.geometry_columns()` instead of using a backend-specific classes, e.g.:
```python
def test_geom_col_name(self):
    try:
        GeoColumn = connection.ops.geometry_columns()
    except NotImplementedError:
        # Not all GIS backends have geometry columns model.
        pass
    else:
        self.assertEqual(
            GeoColumn.geom_col_name(),
            'column_name' if oracle else 'f_geometry_column',
        )
```

Reply:
Thanks for the suggestion.